packages/docprovider/src/yprovider.ts

```diff
@@ -88,8 +95,18 @@ export class WebSocketProvider implements IDocumentProvider {
     this._yWebsocketProvider?.off('connection-close', this._onConnectionClosed);
     this._yWebsocketProvider?.off('sync', this._onSync);
     this._yWebsocketProvider?.destroy();
+    this._disconnect();
     Signal.clearData(this);
   }
+  get sharedModel(): YDocument<DocumentChange> {
+    return this._sharedModel;
+  }
+  setPath(path: string) {
+    this._path = path;
+  }
+  setSharedModel(sharedModel: YDocument<DocumentChange>) {
+    this._sharedModel = sharedModel;
+  }
```

Comment:
```suggestion
```

Is this used anywhere? It would be a significant expansion to the public API, let's keep it small to make it managable to maintain.